keyboards/ducky/one2mini/readme.md

```diff
@@ -0,0 +1,15 @@
+# one2mini
```

Comment:
Please change maintainer to yourself, as well as describing how to enter bootloader, which device models are supported, and how to flash.